attribute-service-api/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ val generateLocalGoGrpcFiles = false
 
 protobuf {
   protoc {
-    artifact = "com.google.protobuf:protoc:3.15.6"
+    artifact = "com.google.protobuf:protoc:3.21.1"
```

Comment:
shall we move to https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java/3.22.0 ?

Reply:
will require code change. Will it be okay to stick to 3.21.12?